docs/release_notes.adoc

```diff
@@ -5,6 +5,22 @@ include::include.adoc[]
 
 == 2.4 (tbd)
 
+=== Breaking Changes
+
+* Calculated responses for interactions (`>> { ... }`) previously were all executed synchronized on the respective mock
```

Comment:
Maybe add a link to the PR here somewhere `spockPull:1910[]`

Reply:
Yeah, always hard to do before the PR is created, as you don't know the number by then. :-D